src/components/tabber/tabber.jsx

```diff
@@ -1,13 +1,36 @@
+import { useQuery } from "@apollo/client";
 import { Tab } from "@headlessui/react";
+import { useContext, useMemo } from "react";
+import { GET_USER_WATCHLIST } from "../../graphql";
+import { CurrentUserContext } from "../../store/current-user-context";
 import { TabList } from "./tab-list";
-import { TabPanel } from "./tab-panel";
 import { TabPanels } from "./tab-panels";
 
-export const Tabber = ({ children }) => {
-  return <Tab.Group>{children}</Tab.Group>;
+export const Tabber = () => {
```

Comment:
This component is used in several places and is meant to be re-usable. We should leave it like it was and then create components in the /routes directory that import the tabber components from here and makes the query for the watchlist, sets up the tab list and panels, etc. Same goes for the tab list and panels-- everything in the /components directory needs to remain re-useable or else we will break other parts of the app that are using this component